<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorFillWidget</class>
 <widget class="QWidget" name="InspectorFillWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutPositionX">
     <item>
      <spacer name="horizontalSpacerPositionX">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>44</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelPositionX">
       <property name="text">
        <string>Position X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderPositionX">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxPositionX"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutPositionY">
     <item>
      <spacer name="horizontalSpacerPositionY">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>44</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelPositionY">
       <property name="text">
        <string>Position Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderPositionY">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxPositionY"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutScaleX">
     <item>
      <spacer name="horizontalSpacerScaleX">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>67</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelScaleX">
       <property name="text">
        <string>Size X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderScaleX">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxScaleX"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutScaleY">
     <item>
      <spacer name="horizontalSpacerScaleY">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>67</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelScaleY">
       <property name="text">
        <string>Size Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderScaleY">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxScaleY"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTransitionDuration">
     <item>
      <spacer name="horizontalSpacerTransitionDuration">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTransitionDuration">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxTransitionDuration">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFrm1">
       <property name="text">
        <string>frm</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTween">
     <item>
      <spacer name="horizontalSpacerTween">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>63</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelEasing">
       <property name="text">
        <string>Tween</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTween">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTriggerOnNext">
     <item>
      <spacer name="horizontalSpacerTriggerOnNext1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>11</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTriggerOnNext">
       <property name="text">
        <string>Trigger on next</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxTriggerOnNext">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerTriggerOnNext2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutUseMipmap">
     <item>
      <spacer name="horizontalSpacerUseMipmap1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>2</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelUseMipmap">
       <property name="text">
        <string>Use Mipmapping</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxUseMipmap">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerUseMipmap2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDefer">
     <item>
      <spacer name="horizontalSpacerDefer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>62</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDefer">
       <property name="text">
        <string>Defer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxDefer">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerDefer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerGeometry">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxTransitionDuration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorFillWidget</receiver>
   <slot>transitionDurationChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTween</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorFillWidget</receiver>
   <slot>tweenChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderPositionX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorFillWidget</receiver>
   <slot>sliderPositionXChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxPositionX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorFillWidget</receiver>
   <slot>spinBoxPositionXChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderPositionY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorFillWidget</receiver>
   <slot>sliderPositionYChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxPositionY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorFillWidget</receiver>
   <slot>spinBoxPositionYChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderScaleX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorFillWidget</receiver>
   <slot>sliderScaleXChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxScaleX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorFillWidget</receiver>
   <slot>spinBoxScaleXChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderScaleY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorFillWidget</receiver>
   <slot>sliderScaleYChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxScaleY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorFillWidget</receiver>
   <slot>spinBoxScaleYChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxDefer</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorFillWidget</receiver>
   <slot>deferChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTriggerOnNext</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorFillWidget</receiver>
   <slot>triggerOnNextChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxUseMipmap</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorFillWidget</receiver>
   <slot>useMipmapChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>scaleXChanged(int)</slot>
  <slot>scaleYChanged(int)</slot>
  <slot>sliderPositionXChanged(int)</slot>
  <slot>sliderPositionYChanged(int)</slot>
  <slot>transitionDurationChanged(int)</slot>
  <slot>tweenChanged(QString)</slot>
  <slot>spinBoxPositionXChanged(int)</slot>
  <slot>spinBoxPositionYChanged(int)</slot>
  <slot>sliderScaleXChanged(int)</slot>
  <slot>sliderScaleYChanged(int)</slot>
  <slot>spinBoxScaleXChanged(int)</slot>
  <slot>spinBoxScaleYChanged(int)</slot>
  <slot>resetDefer(QString)</slot>
  <slot>deferChanged(int)</slot>
  <slot>triggerOnNextChanged(int)</slot>
  <slot>useMipmapChanged(int)</slot>
 </slots>
</ui>
